### ISSUE TITLE
Typo in locales

### ISSUE THREAD

**Issue by eriktelepovsky** [pennersr/django-allauth#3567]
Extra ")" symbol in the translation:

https://github.com/pennersr/django-allauth/blame/f356138d8903d55755a5157e2992d5b40ba93205/allauth/locale/sk/LC_MESSAGES/django.po#L718

```
#: templates/account/messages/password_set.txt:2
msgid "Password successfully set."
msgstr ")Nastavenie hesla bolo úspešné."
```

**Comment by pennersr** [comment]
Fixed via e99fd0e7

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3567
